open_tms_auth/open_tms_auth_be_login/index.js

```diff
@@ -92,20 +93,20 @@ const handler = async ({ req, res }) => {
       is_email_verified: user_account.is_email_verified,
     };
 
-    const secretKey = process.env.BB_OPEN_TMS_SECRET_KEY;
-    const refreshKey = process.env.BB_OPEN_TMS_REFRESH_KEY;
-    console.log("secrete key ", secretKey);
+    const secretKey = process.env.BB_OPEN_TMS_AUTH_SECRET_KEY;
```

Comment:
Make sure you take latest pull from the upstream appblocks_hub/open_tms  repository and sync your fork.We have made some changes to the envs

Reply:
Already made a commit to fix it at [c5a0e93](https://github.com/appblocks-hub/open_tms/pull/19/commits/c5a0e93e0b07a500e59686db564bebf1f54e4c72).

Would probably revert it